tensorflow/lite/kernels/activations.cc

```diff
@@ -937,8 +996,12 @@ TfLiteStatus LeakyReluEval(TfLiteContext* context, TfLiteNode* node) {
           GetTensorShape(output), GetTensorData<float>(output));
       return kTfLiteOk;
     } break;
+    case kTfLiteUInt8: {
+      QLeakyRelu<uint8_t>(input, output, params->alpha, data);
+      return kTfLiteOk;
+    } break;
     default:
-      context->ReportError(context, "Only float32 supported currently, got %s.",
+      context->ReportError(context, "Only float32 and kTfLiteUInt8 supported currently, got %s.",
```

Comment:
lets call this type "uint8" instead of "kTfLiteUInt8" in the error string.

Reply:
@suharshs , thanks for the review, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit